WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -255,7 +402,7 @@ local function createAbilityAndDebuffTrigger(triggers, item)
   createAbilityTrigger(triggers, 1, item, "showAlways");
 end
 
-local function subTypesFor(item)
+local function subTypesFor(item, regionType)
```

Comment:
This function should filter which options are available, so that for models only basic ones are offered. Offering some choices which don't actually work is not great.